rjs/helper.go

```diff
@@ -0,0 +1,38 @@
+package rjs
+
+import "fmt"
+
+// BytesToString converts each byte in a byte slice into character, else panic out
+func BytesToString(lst interface{}) (str string) {
```

Comment:
Is this used anywhere else other than `package rjs` ? If not you dont need to make it exportable, since it's just a helper

Reply:
Actually, is used in `package clients` only.

Reply:
Hence, is required to be exported, if we keep it in `helper.go`